src/gitlab/services/gitlab-api/jobs-api.ts

```diff
@@ -65,8 +65,9 @@ export namespace GitlabJobs {
     ) {
         const projectId = getProjectId();
         const result = new Deferred<JobBase[]>();
+        console.log(projectId, pipelineId);
 
-        $.ajax({ url: `/api/v4/projects/${projectId}/pipelines/${pipelineId}/jobs`, data: filter })
+        $.ajax({ url: `/api/v4/projects/${projectId}/pipelines/${pipelineId}/jobs?per_page=100`, data: filter })
```

Comment:
maybe we could also filter using `scope` parameter to only get success ones? https://docs.gitlab.com/ee/api/jobs.html#list-pipeline-jobs

Reply:
For most pipelines, the majority of jobs will succeed, so the amount of failed jobs will be zero, and even if there is a job or two that failed, excluding them will not help you very much when using the default page limit of 20.

Reply:
Yes, but you're trying to find the first job with given name in the list of jobs. What if the `scope` of the job will be failed? Your script will fail, cuz you're getting the `[0]` job with that name.

Reply:
Ok, by default they are ordered by timestamp desc, but adding it would cause no harm :)